weblate/utils/stats.py

```diff
@@ -833,6 +833,31 @@ def calculate_item(self, item):
         self.store(item, result)
 
 
+class UserStats(BaseStats):
+    def invalidate(self, language: Optional[Language] = None, recurse: bool = True):
+        super().invalidate()
+
+    def _prefetch_basic(self):
+        base = self._object.profile
+        stats = {
+            "all": base.translated + base.suggested + base.uploaded + base.commented,
```

Comment:
Do you see a use case for this? To me it seems like aggregating unrelated metrics...

Reply:
I was thinking as "All Contributions made by user". Maybe doesn't make sense.

Reply:
I think it's easy to aggregate on the client side in case somebody wants it. In case more metrics would be added in the future the meaning of "all" would change in time, what could be confusing.